go-controller/pkg/node/node.go

```diff
@@ -706,8 +706,13 @@ func (n *OvnNode) checkAndDeleteStaleConntrackEntriesForNamespace(newNs *kapi.Na
 		go func(gwIP string) {
 			defer wg.Done()
 			if len(gwIP) > 0 {
+				// TODO: Implement IPv6
```

Comment:
```suggestion
				// TODO: Add support for IPv6 external gateways 
```

Reply:
aren't we supposed to use ipv6 neighbor discovery in this case?

Reply:
we don't support this feature for v6 yet... that can be done in future when the feature is gated.